dateutil/zoneinfo/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 from dateutil.tz import tzfile as _tzfile
 
-__all__ = ["get_zonefile_instance", "gettz", "gettz_db_metadata", "rebuild"]
+__all__ = ["get_zonefile_instance", "gettz", "gettz_db_metadata"]
```

Comment:
I don't think we can change the public interface like this...

Reply:
Hm, actually `rebuild` is a separate file. Yeah, I see how this works. This is fine.

Reply:
the problem here is that this import does not work, the Sphinx could not load it, as no import is there. how is this rebuild suppose to exist?